packages/eslint-plugin/package.json

```diff
@@ -46,7 +46,8 @@
 		"eslint-plugin-react-hooks": "^4.2.0",
 		"globals": "^12.0.0",
 		"prettier": "npm:wp-prettier@2.2.1-beta-1",
-		"requireindex": "^1.2.0"
+		"requireindex": "^1.2.0",
+		"typescript": "^4.1.3"
```

Comment:
I don't think this is a good idea. TypeScript should be a peer dependency if we need to require it. It should be provided by the consumer package and not use the version we require. We don't want duplicates.

If we require packages that have this peer dependency, we should either pass the peer dep along or reconsider whether we can remove that requirement.